packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -37,7 +41,22 @@ export class OrgList implements vscode.Disposable {
 
   private displayTargetOrg(targetOrgOrAlias?: string) {
     if (targetOrgOrAlias) {
-      this.statusBarItem.text = `$(plug) ${targetOrgOrAlias}`;
+      return Promise.resolve(this.isOrgExpired(targetOrgOrAlias))
+        .then(isExpired => {
+          if (isExpired) {
+            this.statusBarItem.text = `$(warning) ${targetOrgOrAlias}`;
```

Comment:
we are showing invalid org as a error but expired org as a warning, why is that?

Reply:
Final? -- Your default org has expired. Some of the command palette commands may no longer work. Switch your default org and try again.

Reply:
Because right now we just want to let users know that it's expired, but we haven't decided what we'll do with expired orgs in general. - We'll decide that after discussing Pete's spike.

It's like the notification on activation if the default org you had set is expired, it's just a warning notification.